Functions/BIDS/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -151,8 +161,16 @@
 		if dimASL(4)>=numel(jsonOut.GELabelingDuration)
 			warning('Labeling duration mismatch with GE private field.');
 			jsonOut.LabelingDuration = jsonOut.GELabelingDuration;
+		elseif dimASL(4)>=numel(unique(jsonOut.GELabelingDuration))
+			warning('Labeling duration mismatch with GE private field.');
+			labDurTemp = unique(jsonOut.GELabelingDuration);
+			if labDurTemp(1) == 0
+				labDurTemp(1:end-1) = labDurTemp(2:end);
+				labDurTemp(end) = 0;
+			end
+			jsonOut.LabelingDuration = labDurTemp;
```

Comment:
`labelingDurationTemp` instead of the abbreviation

Reply:
Fixed in 43d03f7b